koku/cost_models/serializers.py

```diff
@@ -96,20 +101,31 @@ def validate(self, data):
             return data
 
 
+def get_json(path):
```

Comment:
Copy pasted from metrics/views. Would consider making it a utility function instead of writing the same code in 2 places. But more than that in this case we know the JSON file we are trying to load. I would add this kind of error handling if we had user input or different possibly malformed JSON files coming in, but in this case we know it should be a good JSON file so I'd probably swap this entire function with just a 1-line variable definition 

`data = json.load(json_file)`

Reply:
@adberglund is this comment asking for it to eliminate the function entirely?

Reply:
Given what we talked about, what do you think?

Reply:
yes, I think you are asking to eliminate the function entirely.

Reply:
and to replace it with this: https://github.com/project-koku/koku/pull/1986/files#diff-a5fd0aef40ad2b68f502d6854df834b8R51

Reply:
is that correct?